### PR TITLE
fix: typo `isSingnedIn` -> `isSignedIn`

### PR DESCRIPTION
**Pull request by PabloLION**
fix misspelling in [the documentation](https://near.github.io/near-api-js/classes/walletaccount.walletconnection.html#requestsigntransactions)

### CODE REVIEW

File: src/wallet-account.ts [hunk 1]
```diff
@@ -139,32 +140,53 @@ export class WalletConnection {
...
         } else {
             options = contractIdOrOptions as SignInOptions;
         }
 
         const currentUrl = new URL(window.location.href);
         const newUrl = new URL(this._walletBaseUrl + LOGIN_WALLET_URL_SUFFIX);
-        newUrl.searchParams.set('success_url', options.successUrl || currentUrl.href);
-        newUrl.searchParams.set('failure_url', options.failureUrl || currentUrl.href);
-        if (options.contractId) {            
+        newUrl.searchParams.set(
```
Comment (willemneal) [inline src/wallet-account.ts:159]: This formatting should probably be in a different PR.

Comment (PabloLION) [reply src/wallet-account.ts]: fixed in the next commit.

**Review by willemneal — APPROVED** [review]
Thanks!  I have no power here and it seems that the this project is not being maintained at the moment, but if anyone out there who has the power can see this should merge it right away!.

**Comment by PabloLION** [comment]
Thanks.

**Comment by PabloLION** [comment]
@vgrichina Would you mind check the PR? It's just a 3-word change😃

**Review by lewis-sqa — APPROVED** [review]

**Review by amirsaran3 — APPROVED** [review]

**Review by MaximusHaximus — APPROVED** [review]
Thanks for the contribution, and sorry about the delayed response @PabloLION !

**Comment by PabloLION** [comment]
Not a problem. My pleasure to help.
